### ISSUE TITLE
PinchGestureHandler and ColorMatrix won't work

### ISSUE THREAD

**Issue by Yigityakut** [iyegoroff/react-native-color-matrix-image-filters#27]
React Native : 0.61.5
react-native-color-matrix-image-filters: 5.2.2

I have 3 cases. PinchGestureHandler + ColorMatrix together I get an error but both of them working fine when used alone.

Case 1:  This is not working.

```
          <PinchGestureHandler
              onGestureEvent={this._onPinchGestureEvent}
              onHandlerStateChange={this._onPinchHandlerStateChange}>
              <ColorMatrix
                matrix={concatColorMatrices([saturate(10), contrast(0.2)])}>
                <Animated.Image
                  // resizeMode={FastImage.resizeMode.contain}
                  resizeMode="contain"
                  source={{ uri: 'file:///data/user/0/com.reactnativecameratest/cache/Camera/4e215f9b-dc29-430e-b02e-0425ba2dd993.jpg' }}
                  style={{
                    height: screenHeight * 0.65,
                    transform: [
                      { perspective: 15555 },
                      { scale: this._scale },
                    ],
                  }}
                />
              </ColorMatrix>
            </PinchGestureHandler>
```
Case 1 Screenshot:
https://imgur.com/CmyC97M.jpg

Case 2 : Only PinchGestureHandler works fine to zoom the image with pinch gesture

```
            <PinchGestureHandler
              onGestureEvent={this._onPinchGestureEvent}
              onHandlerStateChange={this._onPinchHandlerStateChange}>
              {/* <ColorMatrix matrix={concatColorMatrices([saturate(10), contrast(0.2)])}> */}
                <Animated.Image
                  resizeMode="contain"
                  source={{ uri: 'file:///data/user/0/com.reactnativecameratest/cache/Camera/4e215f9b-dc29-430e-b02e-0425ba2dd993.jpg' }}
                  style={{
                    height: screenHeight * 0.65,
                    transform: [
                      { perspective: 15555 },
                      { scale: this._scale },
                    ],
                  }}
                />
              {/* </ColorMatrix> */}
```

Case 2 Screenshot:
https://imgur.com/jnmbmGq.jpg

Case 3 : Only ColorMatrix works fine with its filters.

```

            {/* <PinchGestureHandler
              onGestureEvent={this._onPinchGestureEvent}
              onHandlerStateChange={this._onPinchHandlerStateChange}> */}
              <ColorMatrix
                matrix={concatColorMatrices([saturate(10), contrast(0.2)])}>
                <Animated.Image
                  // resizeMode={FastImage.resizeMode.contain}
                  resizeMode="contain"
                  source={{ uri: 'file:///data/user/0/com.reactnativecameratest/cache/Camera/4e215f9b-dc29-430e-b02e-0425ba2dd993.jpg' }}
                  style={{
                    height: screenHeight * 0.65,
                    transform: [
                      { perspective: 15555 },
                      { scale: this._scale },
                    ],
                  }}
                />
              </ColorMatrix>
            {/* </PinchGestureHandler> */}
```
Case 3 Screenshot:
https://i.imgur.com/m3TGsZ8.jpg

Logcat log errors: 

```
2020-02-10 02:39:28.266 10548-10619/com.reactnativecameratest E/ReactNativeJS: 'Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?%s%s', '\n\nCheck the render method of `PinchGestureHandler`.', '\n    in ColorMatrix (at App.js:232)\n    in PinchGestureHandler (at App.js:228)\n    in RCTView (at App.js:207)\n    in RCTView (at App.js:131)\n    in App\n    in RCTView (at AppContainer.js:101)\n    in RCTView (at AppContainer.js:119)\n    in AppContainer (at renderApplication.js:39)'
2020-02-10 02:39:28.355 10548-10620/com.reactnativecameratest E/unknown:ReactNative: Exception in native call
    java.lang.NullPointerException: Attempt to invoke virtual method 'double java.lang.Double.doubleValue()' on a null object reference
        at com.facebook.react.bridge.ReadableNativeArray.getDouble(ReadableNativeArray.java:91)
        at com.facebook.react.bridge.JavaMethodWrapper$4.extractArgument(JavaMethodWrapper.java:63)
        at com.facebook.react.bridge.JavaMethodWrapper$4.extractArgument(JavaMethodWrapper.java:59)
        at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:355)
        at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:150)
        at com.facebook.react.bridge.queue.NativeRunnable.run(Native Method)
        at android.os.Handler.handleCallback(Handler.java:789)
        at android.os.Handler.dispatchMessage(Handler.java:98)
        at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:26)
        at android.os.Looper.loop(Looper.java:171)
        at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run(MessageQueueThreadImpl.java:225)
        at java.lang.Thread.run(Thread.java:764)

```

Any ideas how to solve this problem?

**Comment by iyegoroff** [comment]
I guess ColorMatrix component should be wrapped with View or Animated.View.

**Comment by iyegoroff** [comment]
Should be fixed in v5.2.4
